discovery-package/src/main/resources/zosmf-static-definition.yaml.template

```diff
@@ -29,7 +29,7 @@ services:
       description: 'IBM z/OS Management Facility REST API service. Once configured you can access z/OSMF via the API gateway: https://${ZOWE_EXPLORER_HOST}:${GATEWAY_PORT}/ibmzosmf/api/v1/info'
       catalogUiTileId: zosmf
       instanceBaseUrls:
-        - https://${ZOSMF_HOST}:${ZOSMF_PORT}/
+        - ${ZOSMF_SCHEME:-https}://${ZOSMF_HOST}:${ZOSMF_PORT}/
```

Comment:
Do we have a PR to extend schema validation in the `zowe-install-packaging` repo?

Reply:
not yet, I will prepare one